test/functional/services/browser.ts

```diff
@@ -82,6 +82,14 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
         : (driver as any).actions({ bridge: true });
     }
 
+    /**
+     * Get handle for an alert, confirm, or prompt dialog. (if any).
+     * @return {Promise<void>}
+     */
+    public async getAlert() {
+      await driver.switchTo().alert();
```

Comment:
This is going to throw an exception if no alert exists. I think you will need to catch the exception and verify its not NoAlertPresentException

https://stackoverflow.com/questions/11467471/how-to-check-if-an-alert-exists-using-webdriver

```

```